recipes/libplist/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "1.2.240":
+    url: "https://github.com/libimobiledevice-win32/libplist/archive/refs/tags/v1.2.240.tar.gz"
+    sha256: "9de6e5b8cb806e4cde96ef68833fe6baba0094a12072db343e9c2fdd52187e9e"
+  "1.3.17":
+    url: "https://github.com/libimobiledevice-win32/libplist/archive/refs/tags/v1.3.17.tar.gz"
```

Comment:
https://github.com/libimobiledevice-win32/libplist is a fork of https://github.com/libimobiledevice/libplist, which one is official repo now?

Reply:
https://github.com/libimobiledevice/libplist is the officail repo but it is fail to build under windows. https://github.com/libimobiledevice-win32/libplist fix these compile error.

Reply:
Why not provide a patch to upstream instead? You should use Official release: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

Reply:
https://github.com/libimobiledevice/libplist/pull/76 This is the pull request submitted by the owner of https://github.com/libimobiledevice-win32/libplist. It is about 5 years old yet hasn't been merged.

Reply:
Upstream rebased the PR but never answered xD

Reply:
I tried to use official repo and it is failed to build as expected. I also try to make patch for official source, but the change is too much. The patch file generated by git has more than 3000 lines. I am very worried about that this patch will conflict with official new release.